gapis/api/vulkan/transform_mapping_exporter.go

```diff
@@ -178,9 +223,17 @@ func (mappingTransform *mappingExporter) processNotification(ctx context.Context
 			(*mappingTransform.mappings)[replayValue] = make([]service.VulkanHandleMappingItem, 0, 0)
 		}
 
+		if handle.name == "VkFramebuffer" && !mappingTransform.usedFrameBuffers[handle.traceValue] {
+			continue
+		}
+
 		(*mappingTransform.mappings)[replayValue] = append(
 			(*mappingTransform.mappings)[replayValue],
-			service.VulkanHandleMappingItem{HandleType: handle.name, TraceValue: handle.traceValue, ReplayValue: replayValue},
+			service.VulkanHandleMappingItem{
```

Comment:
nit: This looks like a pure formatting change based on adding a trailing comma? I'd recommend to avoid such changes mixed with a functional change as it requires a lot of manual comparing to figure out if anything changed or if this is indeed just a formatting change.

Reply:
This is a leftover, I was trying lots of stuff. I can revert this.

Reply:
Thanks. Still a slight change. More for the future :) I try to remember to do a `git diff` to spot these things to keep the diff as easy as possible to review.